program-runtime/src/loaded_programs.rs

```diff
@@ -86,6 +87,12 @@ pub struct LoadedProgram {
     pub usage_counter: AtomicU64,
 }
 
+#[derive(Debug, Default)]
```

Comment:
Just move `executor::CreateMetrics` from `program-runtime/src/executor.rs` here (including `jit_compile_us`) instead of introducing a new one, though it can still be renamed to `LoadProgramMetrics`.